gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ResourceServiceImpl.java

```diff
@@ -170,4 +218,18 @@ public Completable delete(Resource resource) {
         }
         return Single.just(toValidate);
     }
+
+    private User convert(User user) {
```

Comment:
It seems more a kind of clone or "partial clone" than a convert right?
_When I read it, I first thought it was two different kind of User, then as there's no package name in both, I understood that was more a kind of clone..._

Reply:
FP